doc/stdenv/stdenv.chapter.md

```diff
@@ -899,9 +899,37 @@ This setup hook compresses any man pages that have been installed. The compressi
 
 This runs the strip command on installed binaries and libraries. This removes unnecessary information like debug symbols when they are not needed. This also helps to reduce the installed size of packages.
 
-### `patch-shebangs.sh`
+### `patch-shebangs.sh` {#setup-hook-patch-shebangs}
 
-This setup hook patches installed scripts to use the full path to the shebang interpreter. A shebang interpreter is the first commented line of a script telling the operating system which program will run the script (e.g `#!/bin/bash`). In Nix, we want an exact path to that interpreter to be used. This often replaces `/bin/sh` with a path in the Nix store.
+This setup hook patches installed scripts to use `/nix/store` paths to their shebang interpreter as found in the build environment's `PATH`. The [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) line tells a Unix-like operating system what interpreter to use to execute the script's contents.
```

Comment:
Not necessarily `/nix/store` – although this is most common, store path is configurable.

Reply:
```suggestion
This setup hook patches installed scripts to use Nix store paths to their shebang interpreter as found in the build environment's `PATH`. The [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) line tells a Unix-like operating system what interpreter to use to execute the script's contents.
```

I noticed there is no section in any manual dedicated to the Nix store. It would be useful to have to accommodate some details about it, which are otherwise spread around, and then link to it where necessary.